src/doc/gui_manual/Operators/OperatorTypes/PICS/LCS_operator.rst

```diff
@@ -0,0 +1,183 @@
+.. _LCS operator:
+
+Lagrangian Coherent Structure operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The LCS operator utilizes Lyapunov Exponents based on the Cauchy-Green Tensor
+to highlight LCS's in vector fields. The Cauchy-Green tensor is used as an
+equivalent definition because it is independent of the mesh discretization. As
```

Comment:
@ARSanderson is likely the best source of info on this but I am wondering how much the terminology here (e.g. *Lyapunov Exponents*, *Cauchy-Green Tensor*, *FTLE* (used below) etc.) is useful to users. Also, I think nity grity details like this benefit from adding links to external refs that go into these details with greater rigor than makes sense in a user's manual.

Reply:
> Also, I think nity grity details like this benefit from adding links to external refs that go into these details with greater rigor than makes sense in a user's manual.

Good idea. Then we don't have to keep track of it ourselves

Reply:
If we want to reference the paper listed we should do that. But some of these details are important as there are variations.